packages/gestalt/src/Switch.css

```diff
@@ -4,6 +4,14 @@
   --border: 1px;
 }
 
+html[dir="rtl"] .rtlSupport {
```

Comment:
@selena-zeng Could we just use the `.switch` class? `html[dir="rtl"] .switch {` That way we don't need to add a new class